robottelo/constants/__init__.py

```diff
@@ -1414,6 +1414,11 @@ class Colored(Box):
     'Usergroup': ['view_usergroups', 'create_usergroups', 'edit_usergroups', 'destroy_usergroups'],
 }
 
+PXE_LOADER_MAP = {
+    'bios': {'vm_firmware': 'bios', 'pxe_loader': 'PXELinux BIOS'},
+    'uefi': {'vm_firmware': 'uefi', 'pxe_loader': 'Grub2 UEFI'},
```

Comment:
Instead of defining this here in constants, can this be kept in the `pxe_loader` fixture itself, or is this supposed to be used by other tests in robottelo?

Reply:
Good idea, I do not have a strong opinion about it. I will leave it up to @adarshdubey-star to resolve it eventually.

Reply:
Thanks @Gauravtalreja1 , for now `test_provisioning.py` and `test_discoveredhost.py` are only using this map, moved it to pxe_loader fixture